### PR TITLE
Update kubeadm install page

### PR DESCRIPTION
**Pull request by RA489**
Update kubeadm install page
fixes https://github.com/kubernetes/website/issues/23738

### CODE REVIEW

File: content/fr/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 1]
```diff
@@ -209,9 +209,10 @@ cd /opt/bin
 curl -L --remote-name-all https://storage.googleapis.com/kubernetes-release/release/${RELEASE}/bin/linux/amd64/{kubeadm,kubelet,kubectl}
 chmod +x {kubeadm,kubelet,kubectl}
 
-curl -sSL "https://raw.githubusercontent.com/kubernetes/kubernetes/${RELEASE}/build/debs/kubelet.service" | sed "s:/usr/bin:/opt/bin:g" > /etc/systemd/system/kubelet.service
-mkdir -p /etc/systemd/system/kubelet.service.d
-curl -sSL "https://raw.githubusercontent.com/kubernetes/kubernetes/${RELEASE}/build/debs/10-kubeadm.conf" | sed "s:/usr/bin:/opt/bin:g" > /etc/systemd/system/kubelet.service.d/10-kubeadm.conf
+RELEASE_VERSION="v0.4.0"
```
Comment (neolit123) [inline content/fr/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:212]: this should be the latest - `v0.6.0`
https://github.com/kubernetes/release/releases

Comment (RA489) [reply content/fr/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md]: updated

**Comment by RA489** [comment]
This PR is ready for LGTM.

**Review by neolit123 — COMMENTED** [review]
/lgtm

**Comment by neolit123** [comment]
The pr contains a merge commit so you may have to squash the 2.

**Review by AWKIF — APPROVED** [review]

**Comment by AWKIF** [comment]
i think i validate too quickly, i think the var should be in the bash quote at the same place, line 259
Why did you move it to line 253 ? @RA489

**Pull request merged**